user.php

```diff
@@ -0,0 +1,61 @@
+<?php
+	include("connection.php");
+	/**
+	* @author: mfsi_krishnadev
+	* @class: validating user
+	*/
+	class user extends db_connection
+	{
+		
+		/*
+		 * @access: public
+		 * @param : email-id, password 
+		 * @return type: none
+		*/
+
+		function check($mail_id, $pass)
+		{
+
+			$email_id = $this->test_input($mail_id);
+			$password = md5($pass);
+
+
+			$select = "SELECT id, CONCAT(first_name, ' ',last_name)as name FROM employee
```

Comment:
Mysql keywords should start from new line like below.

$select = "SELECT id, CONCAT(first_name, ' ',last_name)as name 
FROM employee
WHERE email_id = '$email_id' 
AND password = '$password'";

Check everywhere and fix if such code are present.